### PR TITLE
integrity registers, misc wordsmithing 

### PR DESCRIPTION
**Pull request by nedmsmith**
Fixed erroneous wording related to TPMs and added clarity that cross-product equivalence should be applied before applying the `digests` comparison algorithm. Added reference to the cddl that defines what an "identifier" is.

### CODE REVIEW

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

File: draft-ietf-rats-corim.md
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:1871]:
```suggestion
For each Integrity Register entry in the Reference Value, the Verifier will use the associated identifier (i.e., `integrity-register-id-type-choice`) to look up the matching Integrity Register entry in Evidence.
```

File: draft-ietf-rats-corim.md
Comment (thomas-fossati) [inline draft-ietf-rats-corim.md:1873]: also not sure about wording choices:
* "equivalence", and
* "found"
```suggestion
Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}} after the cross-product equivalence has been found according to {{sec-comid-integrity-registers}}.
```

**Review by thomas-fossati — COMMENTED** [review]
LGTM, modulo a bit more wordsmithing

**Review by nedmsmith — COMMENTED** [review]
accepted Thomas's suggestions

**Review by thomas-fossati — APPROVED** [review]
thanks!

File: draft-ietf-rats-corim.md
Comment (yogeshbdeshpande) [inline draft-ietf-rats-corim.md:1873]: Is it possible to simplify this language, before we merge?

Comment (henkbirkholz) [reply draft-ietf-rats-corim.md:1873]: is it necessary? Today is cut-off day. And I am okay enough with the proposal

**Comment by nedmsmith** [comment]
@Yogesh ***@***.***> – can you capture an issue describing the concern with the language?

From: Yogesh Deshpande ***@***.***>
Reply-To: ietf-rats-wg/draft-ietf-rats-corim ***@***.***>
Date: Monday, March 4, 2024 at 8:45 AM
To: ietf-rats-wg/draft-ietf-rats-corim ***@***.***>
Cc: "Smith, Ned" ***@***.***>, Assign ***@***.***>
Subject: Re: [ietf-rats-wg/draft-ietf-rats-corim] integrity registers, misc wordsmithing (PR #200)


@yogeshbdeshpande commented on this pull request.

________________________________

In draft-ietf-rats-corim.md<https://github.com/ietf-rats-wg/draft-ietf-rats-corim/pull/200#discussion_r1511463621>:

>  If no entry is found, the Reference Value does not match.

-Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}}.

-Note that it is not required for all the entries in Evidence to be used during matching: the Reference Value could consist of a "quote" (in TPM parlance) of just a subset of the register space.

+Instead, if an entry is found, the digest comparison proceeds as defined in {{sec-cmp-digests}} after the cross-product equivalence has been found according to {{sec-comid-integrity-registers}}.

Is it possible to simplify this language, before we merge?

File: draft-ietf-rats-corim.md
Comment (yogeshbdeshpande) [reply draft-ietf-rats-corim.md:1873]: I find there is some disconnect in the statement!

the digest comparison proceeds as defined in {{}} after the cross-product equivalence of what?

Do you want to say a set of entries or what ?

Comment (nedmsmith) [reply draft-ietf-rats-corim.md:1873]: Section {{sec-comid-integrity-registers}} describes the process. I didn't want to restate it here. Maybe just remove "cross-product"?

Comment (yogeshbdeshpande) [reply draft-ietf-rats-corim.md:1873]: Works for me! Creates confusion otherwise!

**Comment by nedmsmith** [comment]
See PR #209 
Note that the referenced section uses "cross-product" to describe the process. I'm not changing that wording.

**Pull request merged**
